Actors/VisualObjects/VisualObjects.WebService/wwwroot/Index.html

```diff
@@ -1,4 +1,5 @@
-﻿<html>
+﻿<!DOCTYPE html>
```

Comment:
Other changes look good though, if you revert this index.html change, I'd like to merge this!